README.md

```diff
@@ -1,101 +1,89 @@
+# django-pg-extra-extended (Fork for Django 5+)
+
 <h1 align="center">
   <img width="400" src="https://i.imgur.com/79S6OVM.png" alt="django-postgres-extra">
 </h1>
-  
+
+This is a fork of `django-postgres-extra`, updated to support Django 5+ while maintaining PostgreSQL enhancements for the Django ORM.
```

Comment:
I don't think you mean to commit this.

There are a lot of things in this commit, that you should go over, for starters, CI is broken, can you get that fixed?